README.md

```diff
@@ -66,6 +66,9 @@ fill them in
         $ python api/cfc_webapp.py
 
    You may need to install some of the python dependencies from above
+   Amongst the dependencies include:
+   -cherrypy
```

Comment:
Using the default config, I got this traceback:


Traceback (most recent call last):

  File "api/cfc_webapp.py", line 409, in <module>

    run(host=server_host, port=server_port, server='cherrypy', debug=True)

  File "/Users/jeffdh5/Desktop/e-mission-server/CFC_WebApp/api/bottle.py",
line 2999, in run

    server.run(app)

  File "/Users/jeffdh5/Desktop/e-mission-server/CFC_WebApp/api/bottle.py",
line 2651, in run

    from cherrypy import wsgiserver

ImportError: No module named cherrypy

On Fri, Jan 30, 2015 at 8:02 PM, shankari <notifications@github.com> wrote:

> In README.md
> <https://github.com/e-mission/e-mission-server/pull/65#discussion_r23886250>
> :
>
> > @@ -66,6 +66,9 @@ fill them in
> >          $ python api/cfc_webapp.py
> >
> >     You may need to install some of the python dependencies from above
> > +   Amongst the dependencies include:
> > +   -cherrypy
>
> I believe you shouldn't need cherrypy unless you are running over SSL. Did
> you have an issue with cherrypy when you started with the default config?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/65/files#r23886250>.
>